scripts/mutect2_wdl/unsupported/funcotator.wdl

```diff
@@ -24,135 +24,160 @@
 # this logic *must* go into each task.  Therefore, there is a lot of duplicated code.  This allows users to specify a jar file
 # independent of what is in the docker file.  See the README.md for more info.
 #
+# NOTE: This only does VCF output right now!
+#
 workflow Funcotator {
     String gatk_docker
     File ref_fasta
     File ref_fasta_index
     File ref_dict
     File variant_vcf_to_annotate
+    File variant_vcf_to_annotate_index
     String reference_version
-    String output_file_name
-    String output_format
+    String output_file_base_name
+#    String output_format
 
-    String? data_sources_tar_gz
+    File? interval_list
```

Comment:
Need to add a comment up top for `interval_list` parameter.  Make sure to mention that it is optional.